akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst

```diff
@@ -30,21 +30,11 @@ in practice. No changes were needed to the Akka source code for this update. Use
 depend on 3.8.0.Final that break with 3.10.3.Final should be able to manually downgrade the dependency
 to 3.8.0.Final and Akka will still work with that version.
 
-Added parameter validation to RootActorPath
```

Comment:
Only restructuring in this file, except for the new section "PersistentView is deprecated"